common_test/src/main/java/uk/gov/dhsc/htbhf/eligibility/model/testhelper/CombinedIdAndEligibilityResponseTestDataFactory.java

```diff
@@ -87,6 +87,57 @@ public static CombinedIdentityAndEligibilityResponse anIdMatchedEligibilityConfi
                 .build();
     }
 
+    public static CombinedIdentityAndEligibilityResponse anIdMatchedEligibilityConfirmedEmailNotMatchedResponse() {
```

Comment:
Rather than creating these methods, I have used the method `anIdMatchedEligibilityConfirmedUCResponseWithMatches` which allows you to customise and pick the email and phone verification responses you want.

Reply:
I think these methods read nicer as they're much shorter than having to pass in three parameters.

Reply:
In which case these new methods should probably call 
```
anIdMatchedEligibilityConfirmedUCResponseWithMatches(VerificationOutcome mobileVerification,
                                                                                                              VerificationOutcome emailVerification,
                                                                                                              List<LocalDate> childrenDobs) {
```
rather than set up the response themselves

Reply:
Updated so the new methods call anIdMatchedEligibilityConfirmedUCResponseWithMatches